pkg/kubelet/cm/qos_container_manager_linux.go

```diff
@@ -209,12 +236,17 @@ func (m *qosContainerManagerImpl) UpdateReserve() error {
 		},
 	}
 
+	if err := m.setCPUReserve(qosConfigs); err != nil {
```

Comment:
@sjenning -- we need to run a reconciliation loop to ensure that BestEffort is always 2 shares in case anyone else ever mutates it on the host w/o kubelet knowledge.